subcommittees/mentoring/procedure-debriefing-session.md

```diff
@@ -0,0 +1,93 @@
+# Procedure - Debriefing Session
+
+This document intends to describe how debriefing sessions
+are organized so that it continue even after members go away.
+
+## Terminology
+
+-   **Debriefing session**: is an online meeting
+    where instructors could share experience about the workshops
+    they taught.
```

Comment:
Yes, that is very good clarification (maybe even add if possible exact time"taught in the last x weeks").